src/EditorFeatures/CSharpTest/CodeActions/IntroduceVariable/IntroduceVariableTests.cs

```diff
@@ -1661,7 +1661,18 @@ void Main()
 #line default
         Goo([|1 + 1|]);
     }
-}", new TestParameters(Options.Regular));
+}",
+@"class Program
+{
+#line hidden
+    void Main()
+    {
+#line default
+        Goo(V);
+    }
+
+    private const int {|Rename:V|} = 1 + 1;
+}", parameters: new TestParameters(Options.Regular));
```

Comment:
this is an improvement over our previous behavior (while also being faster!).  We have a reasonable location where we can place the constant.  So we shouldn't suppress the feature.